elasticjob-lite/elasticjob-lite-core/pom.xml

```diff
@@ -125,5 +121,11 @@
             <artifactId>logback-classic</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.shardingsphere.elasticjob</groupId>
+            <artifactId>elasticjob-registry-center-zookeeper-curator</artifactId>
+            <version>${project.parent.version}</version>
+            <scope>test</scope>
```

Comment:
Could we consider setting it `runtime`? Otherwise users need to introduce it sperately.

Reply:
Yes, after decoupling, users are expected to actively introduce.